app/views/schedules/_event.html.haml

```diff
@@ -33,6 +33,22 @@
 
 :javascript
 
+  /* Links inside event-panel (to make ctrl + click work for these links):
+   = link_to text, '#', onClick: 'insideLinkClicked();', 'data-url' => url
+  */
+  function insideLinkClicked(e){
```

Comment:
It is not called anywhere because there is not any link inside event-panel apart from the `more` link, which is the same link than the event-panel one, so it is not necessary to use it. But @nishanthvijayan is going to add a link inside event-panel in https://github.com/openSUSE/osem/pull/1113.

Then if we merge this he can use the function, or if you prefer to merge https://github.com/openSUSE/osem/pull/1113 first, I'll addapt this PR afterwards to use this function in the new link.